notebooks/official/matching_engine/sdk_matching_engine_create_multimodal_embeddings.ipynb

```diff
@@ -0,0 +1,1533 @@
+{
```

Comment:
<p>I think I'd add the link as a hyperlink. Something like - "..to your service account in the iam gsutil command official documentation" (and link the "iam gsutil command")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Ditto.</p>